### DISCUSSION TITLE
Amount on payment page not as set in json encode Flutter(Flutter code)

### DISCUSSION THREAD

**Discussion by airstarcc** [flutter-stripe/flutter_stripe#1226] [Q&A]
Hi. Need your help regarding this issue. This is first time i am using this package. Thank you.

**Describe the bug**
Amount on payment page not as set in json code flutter. Dont know how to set the amount to be same as json in flutter code. in Flutter code, the amount i set is MYR 50.00 not MYR 3.50

**To Reproduce**
Flutter Code:
`    final url = Uri.parse('$kApiUrl/create-payment-intent');
    final response = await http.post(
      url,
      headers: {
        'Content-Type': 'application/json',
      },
      body: json.encode({
        'currency': 'myr',
        'payment_method_types': ['fpx'],
        "amount": 5000,
      }),
    );
`

Server code:
```
const itemIdToPrice: { [id: string]: number } = {
  'id-1': 350,
  'id-2': 350,
  'id-3': 350,
  'id-4': 350,
  'id-5': 350,
};

const calculateOrderAmount = (itemIds: string[] = ['id-1']): number => {
  const total = itemIds
    .map((id) => itemIdToPrice[id])
    .reduce((prev, curr) => prev + curr, 0);

  return total;
};
```

**Expected behavior**
Amount on server code should be same as the amount from Flutter code.

**Smartphone**
 - Device: Samsung A50
 - OS: Android 11
 - Package version: Latest
 - Flutter version 3.7.12

**Comment by remonh87** [comment]
Hi, the server code is just for testing and is not a full fletched API. We use this purely for debugging on our side. It is a copy of the stripe example backend.

**Reply by brandonLYQ00** [reply]
Hi there @remonh87, i have a question regarding the $kApiUrl, may i know if it is different for each user in the config.dart? If yes, where do i go to get my own values

**Reply by remonh87** [reply]
the variable is just the endpoint where you serve the backend. if you use the example backend (for testing) it is localhost:4242
